cmd/nvidia-cdi-hook/main.go

```diff
@@ -44,52 +44,37 @@ func main() {
 	// Create a options struct to hold the parsed environment variables or command line flags
 	opts := options{}
 
-	// Create the top-level CLI
-	c := cli.Command{
```

Comment:
Does it make sense to still construct the `Command`s and then call a fuction to modify the members that we want instead?